include/aie/Dialect/AIE/IR/AIEOps.td

```diff
@@ -891,7 +891,7 @@ def AIE_DMAStartOp: AIE_Op<"dma_start", [
   );
   let successors = (successor AnySuccessor:$dest, AnySuccessor:$chain);
   let assemblyFormat = [{
-    `(` $channel_dir `,` $channel_index `,` $dest `,` $chain (`,` `repeat_count` `=` $repeat_count^)? `)` attr-dict
```

Comment:
This change means that the MLIR syntax will change from
`aie.dma_start("MM2S", 0, ^bd0, ^end, repeat_count=2)`
to 
`aie.dma_start("MM2S", 0, ^bd0, ^end) {repeat_count=2}`
right?

Reply:
There's a reason I'm doing (that's important) and I will discuss before I land

Reply:
Sounds good! :)